core/benches/banking_stage.rs

```diff
@@ -168,9 +168,9 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
     let (tpu_vote_sender, tpu_vote_receiver) = unbounded();
     let (vote_sender, vote_receiver) = unbounded();
     let mut bank = Bank::new_for_benches(&genesis_config);
+    bank.ns_per_slot = u128::MAX;
+    let bank = Arc::new(bank);
     // Allow arbitrary transaction processing time for the purposes of this bench
```

Comment:
A little bit of a nitpick, but our convention is to place comments before the logic they are explaining as opposed to after. So, could you please keep
```
    // Allow arbitrary transaction processing time for the purposes of this bench
```
at the line before
```
    bank.ns_per_slot = u128::MAX;
```